cli/src/index.ts

```diff
@@ -25,4 +26,11 @@ program
         runGenerate(options.pattern, options.output, !!options.verbose);
     });
 
+program
+    .command('validate')
+    .requiredOption('-p, --pattern <pattern>', 'The location of the JSON Schema Pattern')
+    .requiredOption('-i, --instantiation <instantiation>', 'The location of the instantiation of the Pattern')
+    .option('-m, --metaSchemasLocation <metaSchemaLocation>', 'The location of the directory of the meta schemas to be loaded', '../calm/draft/2024-03/meta')
```

Comment:
not sure about this as a default value. It will work while we are working on this locally, but in the bundled version that folder won't exist. Granted, we haven't done the work to bundle it yet. We may just have to remember to change this later on.

Reply:
There are a few things on this PR that we want to change down the line, but we just want to get this merged so people have something to validate with in the meantime.